packages/remix-forms/src/createForm.tsx

```diff
@@ -333,34 +345,45 @@ function createForm({
       } as Field<SchemaType>
     }
 
-    const hiddenFieldsErrorsToGlobal = (globalErrors: string[] = []) => {
-      const deepHiddenFieldsErrors = hiddenFields?.map((hiddenField) => {
-        const hiddenFieldErrors = fieldErrors(hiddenField)
-
-        if (hiddenFieldErrors instanceof Array) {
-          const hiddenFieldLabel =
-            (labels && labels[hiddenField]) || inferLabel(String(hiddenField))
-          return hiddenFieldErrors.map(
-            (error) => `${hiddenFieldLabel}: ${error}`,
-          )
-        } else return []
-      })
-      const hiddenFieldsErrors: string[] = deepHiddenFieldsErrors?.flat() || []
+    const hiddenFieldsErrorsToGlobal = React.useCallback(
```

Comment:
Is it worth to add this `useCallback`? I'm wary of trying to gain uncertain performance gain for certain complexity increase.